src/Microsoft.ManagedZLib/benchmarks/ManagedZLibBenchmarks.cs

```diff
@@ -17,34 +20,58 @@ public class ManagedZLibBenchmark
 {
     public static IEnumerable<string> UncompressedTestFileNames()
     {
-        yield return "TestDocument.pdf"; // 199 KB small test document with repeated paragraph, PDF are common
-        yield return "alice29.txt"; // 145 KB, copy of "ALICE'S ADVENTURES IN WONDERLAND" book, an example of text file
+        //yield return "TestDocument.pdf"; // 199 KB small test document with repeated paragraph, PDF are common
+        //yield return "alice29.txt"; // 145 KB, copy of "ALICE'S ADVENTURES IN WONDERLAND" book, an example of text file
         yield return "sum"; // 37.3 KB, some binary content, an example of binary file
     }
 
+    [ParamsSource(nameof(UncompressedTestFileNames))]
+    public string? Files { get; set; }
+
+    [Params(System.IO.Compression.CompressionLevel.SmallestSize,
+            System.IO.Compression.CompressionLevel.Optimal,
+            System.IO.Compression.CompressionLevel.Fastest)] // we don't test the performance of CompressionLevel.NoCompression on purpose
+    public System.IO.Compression.CompressionLevel Level { get; set; }
+
+    [Params(CompressionLevel.SmallestSize,
+            CompressionLevel.Optimal,
+            CompressionLevel.Fastest)] // we don't test the performance of CompressionLevel.NoCompression on purpose
+    public CompressionLevel ManagedLevel { get; set; }
+
+    public byte[]? UncompressedData { get; set; }
+    public byte[]? CompressedData { get; set; }
+
+    public MemoryStream? CompressedStreamN { get; set; }
+    public MemoryStream? CompressedStreamM { get; set; }
+
+    DeflateStream? compressionStreamM;
+    System.IO.Compression.DeflateStream? compressionStreamN;
```

Comment:
Rename these, since they are fields (here and all the references). And I assume they can be private:

```suggestion
    private DeflateStream? _compressionStreamM;
    private System.IO.Compression.DeflateStream? _compressionStreamN;
```